src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -426,9 +424,9 @@ export class EditDataGridPanel extends GridParentComponent {
 			clearTimeout(self.refreshGridTimeoutHandle);
 			this.refreshGridTimeoutHandle = setTimeout(() => {
 
-				if (self.dataSet && self.placeHolderDataSets[0].resized) {
+				if (self.dataSet) {
 					self.placeHolderDataSets[0].dataRows = self.dataSet.dataRows;
-					self.placeHolderDataSets[0].resized.fire();
+					setTimeout(() => this.onResize(), 100);
```

Comment:
Is this supposed to replace the throttle? That isn't what's going to happen here - it'll just delay the call by 100ms but if this function is called multiple times in a row then each of those will queue up and fire separately. 

(There's some extra consideration here for how we're handling the refresh grid batching/throttling, but in general I don't see why the timeout here is doing anything useful. Especially if it's supposed to replace the throttle call.)